js/lib/urlutil.js

```diff
@@ -429,11 +429,14 @@ const UrlUtil = {
     }
 
     let parsed = urlParse(location)
+    if (parsed.origin) {
```

Comment:
I think at some point in the near future we should convert to chromium urls with a trailing `/`. Removing them for backwards compat is sure to create problems in the long-term